ox_profile/core/recording.py

```diff
@@ -1,33 +1,13 @@
-"""Module for recording and saving measuremnts.
+"""Module for recording and saving measurements.
 """
 
 import re
 import threading
+from collections import defaultdict, namedtuple, Counter
 
 
-class ProfileRecord(object):
-    """Simple record to track how many times a function/path is called.
-    """
-
-    def __init__(self, name, hits):
-        """Initializer.
-
-        :param name:   String name of function or stack path.
-
-        :param hits:   Number of times it is called.
-        """
-        self.name = name
-        self.hits = hits
-
-    def to_str(self):
-        "Return string reprsentation."
-        result = '%s(%s=%s, %s=%s)' % (
-            self.__class__.__name__, 'name', self.name, 'hits', self.hits)
-
-        return result
-
-    def __repr__(self):
-        return self.to_str()
+# Simple record to track how many times a function/path is called.
+ProfileRecord = namedtuple('ProfileRecord', 'name, hits')
```

Comment:
I like namedtuple as well but want to be able to update fields in ProfileRecord so prefer to keep it as a class not tuple. Please revert those changes.

Reply:
Ok, I've reverted that class back. I was trying to remove unnecessary overhead of creating such a simple object. Also I'm not sure why do we need that dedicated "to_str()" method... But if you are going to add some functionality to the "ProfileRecord" later - I'm ok with that.